src/Halogen/HTML/CSS.purs

```diff
@@ -64,8 +65,8 @@ style =
   rights = concatMap $ foldMap singleton
 
 -- | Render a set of rules as a `style` element.
-stylesheet ∷ ∀ p i. CSS → HTML p i
+stylesheet ∷ ∀ p i. CSS → HC.HTML p i
```

Comment:
Yeah, that was my initial intention. But `Node` and `Leaf` from `Halogen.HTML.Elements` are functions from `Array (IProp _)` now and this thing doesn't use any props